argusclient/client.py

```diff
@@ -591,6 +591,198 @@ def get_alerts_allinfo(self, ownerName=None, alertNameContains=None, shared=Fals
+
+        uri_path = "alerts/{}/triggers".format(alert_id)
+        return self.argus._request("post", uri_path, dataObj=trigger)
+
+    def del_trigger_from_child_alert(self, alert_id, trigger_id):
+        """
+        Delete child alert from a composite alert
+
+        :param comp_alert_id: ID of a composite alert
+        :type comp_alert_id: Integer
+
+        :param trigger_id: ID of a trigger belonging to child alert
+        :type trigger_id: Integer
+        """
+        if not alert_id: raise ValueError("Need to specify an alert id")
+        if not trigger_id: raise ValueError("Need to specify a trigger id")
+        if not isinstance(alert_id, int): raise TypeError("Need an Alert ID, got: %s" % type(alert_id))
+        if not isinstance(trigger_id, int): raise TypeError("Need a trigger id, got: %s" % type(trigger_id))
+
+        uri_path = "alerts/{}/triggers/{}".format(alert_id, trigger_id)
+        return self.argus._request("delete", uri_path)
+        
+    def add_notification_to_composite_alert(self, comp_alert, notification):
+        """
+        Add notification to composite alert
+
+        :param comp_alert: composite alert object
+        :type comp_alert: class:`argusclient.model.Alert`
+
+        :param trigger: Notification object to be added to a composite alert 
+        :type child_alert_id: class:`argusclient.model.Notification`
+
+        :return: Newly created notification object of type class:`argusclient.model.Notification`
+        """
+        if not comp_alert: raise ValueError("Need to specify Alert object")
+        if not notification: raise ValueError("Need to specify Notification object")
+        if not isinstance(comp_alert, Alert): raise TypeError("Need an Alert object, got: %s" % type(comp_alert))
+        if not isinstance(notification, Notification): raise TypeError(
+            "Need a Notification object, got: %s" % type(notification))
+        
+        notification_obj = comp_alert.notifications.add(notification)
+        notification_obj.notification = notification_obj
+        return notification_obj
+
+    def del_notification_from_composite_alert(self, comp_alert, notification_id):
+        """
+        Delete notification from a composite alert
+        
+        :param comp_alert: composite alert object
+        :type comp_alert: class:`argusclient.model.Alert`
+
+        :param notification_id: ID of a notification belonging to composite alert
+        :type notification_id: Integer
+
+        """
+        if not comp_alert: raise ValueError("Need to specify an Alert ")
+        if not notification_id: raise ValueError("Need to specify a notification id")
+        if not isinstance(comp_alert, Alert): raise TypeError("Need an Alert object, got: %s" % type(comp_alert))
+        if not isinstance(notification_id, int): raise TypeError("Need a Notifications id, got: %s" % type(notification_id))
+        
+        return comp_alert.notifications.delete(notification_id)
```

Comment:
Why can't the user do this directly? What value is this wrapper method adding? That is what I meant by calling this a duplicate method in my earlier comment here: https://github.com/salesforce/python-argusclient/pull/28#discussion_r615901002

Reply:
Removed the wrapper function